### PR TITLE
some documentation updates

### PR DESCRIPTION
**Pull request by poettering**
Fixes for #6097 and #5696.

### CODE REVIEW

File: man/systemd.unit.xml [hunk 1]
```diff
@@ -980,13 +980,11 @@
...
-        on whether the system is booting up with an unpopulated
-        <filename>/etc</filename> directory. This may be used to
-        populate <filename>/etc</filename> on the first boot after
-        factory reset, or when a new system instances boots up for the
-        first time.</para>
+        <para><varname>ConditionFirstBoot=</varname> takes a boolean argument. This condition may be used to
+        conditionalize units on whether the system is booting up with an unpopulated <filename>/etc</filename>
+        directory (specifically: an <filename>/etc</filename> with no <filename>/etc/machine-id</filename>). This may
+        be used to populate <filename>/etc</filename> on the first boot after factory reset, or when a new system
+        instances boots up for the first time.</para>
```
Comment (keszybz) [inline man/systemd.unit.xml]: There's plural / singular mismatch in that last sentence.

File: man/systemd.service.xml [hunk 2]
```diff
@@ -328,14 +328,12 @@
...
-        (i.e. the process has been started for <varname>Type=simple</varname>
-        or <varname>Type=idle</varname>, the process exits successfully for
-        <varname>Type=oneshot</varname>, the initial process exits successfully
-        for <varname>Type=forking</varname>, <literal>READY=1</literal> is sent
-        for <varname>Type=notify</varname>, or the <varname>BusName=</varname>
-        has been taken for <varname>Type=dbus</varname>).</para>
+        <para><varname>ExecStartPost=</varname> commands are only run after the commands specified in
+        <varname>ExecStart=</varname> have been invoked successfully, as determined by <varname>Type=</varname>
+        (i.e. the process has been started for <varname>Type=simple</varname> or <varname>Type=idle</varname>, the
+        process exited successfully for <varname>Type=oneshot</varname>, the initial process exited successfully for
```
Comment (keszybz) [inline man/systemd.service.xml]: ... the process *for the last ExecStart= command* has exited successfully for Type=simple ... ?

File: man/systemd.unit.xml [hunk 3]
```diff
@@ -546,26 +546,29 @@
...
-        is shut down and the other is started up. The shutdown is ordered before the start-up in all cases. If two
-        units have no ordering dependencies between them, they are shut down or started up simultaneously, and no
-        ordering takes place.  </para></listitem>
+        <listitem><para>These two settings expect a space-separated list of unit names. They configure ordering
+        dependencies between units. If a unit <filename>foo.service</filename> contains a setting
+        <option>Before=bar.service</option> and both units are being started, <filename>bar.service</filename>'s
+        start-up is delayed until <filename>foo.service</filename> has finished starting up.  Note that this setting is
+        independent of and orthogonal to the requirement dependencies as configured by <varname>Requires=</varname>,
+        <varname>Wants=</varname> or <varname>BindsTo=</varname>. It is a common pattern to include a unit name in both
+        the <varname>After=</varname> and <varname>Requires=</varname> option, in which case the unit listed will be
```
Comment (keszybz) [inline man/systemd.unit.xml]: option*s*

File: man/systemd.unit.xml [hunk 4]
```diff
@@ -546,26 +546,29 @@
...
+        started before the unit that is configured with these options. This option may be specified more than once, in
+        which case ordering dependencies for all listed names are created. <varname>After=</varname> is the inverse of
+        <varname>Before=</varname>, i.e. while <varname>After=</varname> ensures that the configured unit is started
+        after the listed unit finished starting up, <varname>Before=</varname> ensures the opposite, i.e. that the
```
Comment (keszybz) [inline man/systemd.unit.xml]: <strike>i.e.</strike> (it's the second time in that sentence, just doesn't sound good)

File: man/systemd.service.xml (hunk 2 above)
Comment (poettering) [reply man/systemd.service.xml]: I figure you mean Type=oneshot here?

**Comment by poettering** [comment]
Fixed all issues you pointed out, PTAL

**Pull request merged**